crates/app/src/main.rs

```diff
@@ -54,5 +55,8 @@ async fn main() -> anyhow::Result<()> {
         .await
         .unwrap();
 
+    let task_repo = DomainTaskRepository::new(pool.clone());
+    // let tasks = task_repo.get_tasks().await?;
+
```

Comment:
We may want to move this into `service_registry.rs`.

Reply:
okie